web/src/components/AddPlayers.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <div class="players">
+    <div class="players__list">
+      <table class="table table-hover">
+        <thead>
+        <tr>
+          <th>#</th>
+          <th>First Name</th>
+          <th>Last Name</th>
+          <th>Actions</th>
+        </tr>
+        </thead>
+        <tbody name="table-row">
+        <tr v-for="player in players" :key="player.id">
+          <td :key="player.id">
+          <td>{{player.firstname}}</td>
+          <td>{{player.lastname}}</td>
+          <td>
+            <button v-on:click="deletePlayer(player)">Delete</button>
+            <br/>
+            <button>Edit </button>
+            <br/>
+          </td>
+        </tr>
+        </tbody>
+      </table>
+    </div>
+    <div class="players_form_container">
+      <form id="form" v-on:submit.prevent="addPlayer" class="players_form_container__form">
+        <input type="text" v-model="newPlayer.firstname" placeholder="Firstname">
+        <input type="text" v-model="newPlayer.lastname" placeholder="Lastname">
+        <input type="submit" value="Save">
+      </form>
+    </div>
+    <!-- errory tez by dobrze było zrobić XD -->
+    <!--<ul class="errors">-->
+    <!--<li v-show="!validation.name">Name cannot be empty.</li>-->
+    <!--<li v-show="!validation.email">Please provide a valid email address.</li>-->
+    <!--</ul>-->
+  </div>
+</template>
+
+<script>
+  import axios from 'axios'
+
+  export default {
+    name: 'AddPlayers',
+    // data used in template
+    data: function () {
+      return {
+        players: [],
+        newPlayer: {}
+      }
+    },
+    // initial data load
+    created: function () {
+      axios.get(this.$config.API + 'players').then(response => {
+        this.players = response.data
+      }).catch(e => {
+        throw e
+      })
+    },
+    // methods used in template
+    methods: {
+      addPlayer: function () {
+        let self = this
+        axios.post(this.$config.API + 'players', this.newPlayer).then(response => {
+          console.log(self.players);
+          self.players.push(this.newPlayer)
+          console.log(self.players);
+          self.newPlayer = {}
+        }).catch(e => {
+          throw e
```

Comment:
Jak wyżej

Reply:
no tutaj powinniśmy ustawić self.error, które później wyrenderuje jakiś error w tym formie wyżej :P Nie doszedłem do tego szczerze mówiąc, więc wrzuciłem to co mam :P

Reply:
okej, olałbym na razie jak starczy czasu to ogarniemy :)